src/components/HomePageCards/HomePageCards.js

```diff
@@ -27,7 +27,9 @@ export const HomePageCards = () => {
           </CardBody>
 
           <CardFooter>
-            <RouterLink to="/documentation">Documentation</RouterLink>
+            <RouterLink to="/documentation" target={"_blank"}>
```

Comment:
I'm not sure we want target= "_blank" since it opens in a new tab. It sort of breaks the flow when it opens the pages in a new tab when they are from the same app.